io/disk/iozone.py

```diff
@@ -0,0 +1,649 @@
+import os
```

Comment:
PS: I noticed we forgot doing this in all other tests, I'll take care of them and I'd like you to take care of this one.